go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/argoproj/argo-rollouts v1.4.1
 	github.com/cenkalti/backoff/v4 v4.2.1
 	github.com/cloudevents/sdk-go/v2 v2.14.0
-	github.com/dapr/components-contrib v1.13.0-rc.1
+	github.com/dapr/components-contrib v1.13.0-rc.2
```

Comment:
v1.13.0-rc.2 is no more. Long live `v1.13.0-rc.3`

Reply:
Updated